{"uuid": "a7037ede-6dd5-48b9-a2d3-3680058cd272", "children": ["de855524-5906-4c44-b7df-7d3aaf6c18d4"], "befores": [{"name": "_xunit_setup_method_fixture_TestUserAuth", "status": "broken", "statusDetails": {"message": "requests.exceptions.MissingSchema: Invalid URL 'ENV_OBJECT.get_base_url(/user/login': No scheme supplied. Perhaps you meant http://ENV_OBJECT.get_base_url(/user/login?\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/_pytest/python.py\", line 893, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/_pytest/python.py\", line 778, in _call_with_optional_argument\n    func()\n  File \"/Users/pavlovairisa/PycharmProjects/Rest_Api_Python/tests/test_user_auth.py\", line 18, in setup\n    response1 = MyRequests.post(\"/user/login\",data=data)\n  File \"/Users/pavlovairisa/PycharmProjects/Rest_Api_Python/lib/my_requests.py\", line 10, in post\n    return MyRequests._send(url, data, headers, cookies, 'POST')\n  File \"/Users/pavlovairisa/PycharmProjects/Rest_Api_Python/lib/my_requests.py\", line 41, in _send\n    response= requests.post(url, data=data, headers=headers, cookies=cookies)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/sessions.py\", line 573, in request\n    prep = self.prepare_request(req)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/sessions.py\", line 484, in prepare_request\n    p.prepare(\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/models.py\", line 368, in prepare\n    self.prepare_url(url, params)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/models.py\", line 439, in prepare_url\n    raise MissingSchema(\n"}, "steps": [{"name": "POST request to URL '/user/login'", "status": "broken", "statusDetails": {"message": "requests.exceptions.MissingSchema: Invalid URL 'ENV_OBJECT.get_base_url(/user/login': No scheme supplied. Perhaps you meant http://ENV_OBJECT.get_base_url(/user/login?\n", "trace": "  File \"/Users/pavlovairisa/PycharmProjects/Rest_Api_Python/lib/my_requests.py\", line 10, in post\n    return MyRequests._send(url, data, headers, cookies, 'POST')\n  File \"/Users/pavlovairisa/PycharmProjects/Rest_Api_Python/lib/my_requests.py\", line 41, in _send\n    response= requests.post(url, data=data, headers=headers, cookies=cookies)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/sessions.py\", line 573, in request\n    prep = self.prepare_request(req)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/sessions.py\", line 484, in prepare_request\n    p.prepare(\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/models.py\", line 368, in prepare\n    self.prepare_url(url, params)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/requests/models.py\", line 439, in prepare_url\n    raise MissingSchema(\n"}, "start": 1660156990076, "stop": 1660156990076}], "start": 1660156990076, "stop": 1660156990076}], "start": 1660156990076, "stop": 1660156990114}